examples/triton_gpt2/README.ipynb

```diff
@@ -9,17 +9,20 @@
     "\n",
```

Comment:
<p>Would it make sense to run <code>black</code>  over this snippet and the previous one? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/seldon-core/pull/3217/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Reply:
<p>It makes sense - ran it</p><p> </p>